src/main/java/org/pac4j/demo/j2e/ui/ProfileView.java

```diff
@@ -55,10 +54,41 @@ public List getProfiles() {
     }
 
 
+    /**
+     * Gets the Google OIDC profile if it exists.
+     *
+     * @return the Google OIDC profile
+     */
+    public GoogleOidcProfile getGoogleOidcProfile() {
```

Comment:
Yes, you're correct, and I had intended to do that, but I ran into a few problems with dereferencing in the facelets files.  Either it's not possible to specify a class literal in the JSF expression language, or I haven't found the proper way to do it yet.  A workaround I'm trying is to have a `getProfile(String)` which the facelets files can use to invoke with a String expression of the classname, and then this method can lookup the class and invoke the `getProfile(Class)` if the class is found.  I'll submit another pull request after I make these changes.